modules/aaudio/player.c

```diff
@@ -0,0 +1,163 @@
+/**
+ * @file aaudio/player.c  AAudio audio driver for Android
+ *
+ * Copyright (C) 2024 Juha Heinanen
+ * Copyright (C) 2024 Sebastian Reimers
+ */
+
+#include <re.h>
+#include <rem.h>
+#include <baresip.h>
+
+#include "aaudio.h"
+
+
+struct auplay_st {
+	struct auplay_prm play_prm;
+	auplay_write_h *wh;
+	size_t sampsz;
+	void *arg;
```

Comment:
I would suggest to group the write-handler and arg together, they usually go in a pair